src/test/kotlin/three/consulting/epoc/repository/CustomerServiceIntegrationTest.kt

```diff
@@ -27,4 +27,26 @@ class CustomerServiceIntegrationTest : IntegrationTest() {
         val customer: CustomerDTO? = customerService.findCustomerForId(1000L)
         assertThat(customer).isNull()
     }
+
+    @Test
+    fun `added customer is found from database`() {
+        assertThat(false)
+        val customer = CustomerDTO(null,"Testit Oy", "Innovating new innovation!", enabled = true)
+        val addedCustomer: CustomerDTO? = customerService.createCustomer(customer)
+        if (addedCustomer != null) {
+            if (addedCustomer.id != null) {
+                val dbCustomer: CustomerDTO? = customerService.findCustomerForId(addedCustomer.id!!)
+                assertThat(addedCustomer).isEqualTo(dbCustomer)
+                assertThat(addedCustomer.created).isNotNull
+            }
+            else throw RuntimeException()
+        }
+        else throw RuntimeException()
+    }
+    @Test
+    fun `adding customer with id fails`() {
+        assertThat(false)
```

Comment:
Same as above, shouldn't be here.

Reply:
Just came to my mind that the way to test that the service method fails expectedly and throws a type of error would be to use `assertThatThrownBy` from the assertj core library like
```kotlin
assertThatThrownBy { customerService.createCustomer(someInvalidCustomerObject) }
    .isInstanceOf(UnableToCreateCustomerException::class.java)
    .hasMessage("Cannot create a customer with existing id")
```